docs/BuildSource.md

```diff
@@ -70,3 +70,8 @@ cmake --build ./build --config Release --target install
 # Alternatively, build in Visual Studio via its built-in support for
 # CMake or by opening the generated build/timescaledb.sln solution file.
 ```
+
+#### Older PostgreSQL versions
+TimescaleDB can be installed with PostgreSQL 9.6.6+ and 10.9+ versions. Supprot for these 
```

Comment:
```suggestion
TimescaleDB can be installed with PostgreSQL 9.6.6+ and 10.9+ versions. Support for these 
```

Reply:
Thank you. Found a setting to use spelling in the editor. Fixed